Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -525,30 +525,59 @@ private bool IsLobbyNotificationValid(NotifyEventHandle handle)
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
         private ulong AddNotifyLobbyUpdateReceived(LobbyInterface lobbyInterface, OnLobbyUpdateReceivedCallback notificationFn)
         {
             var options = new AddNotifyLobbyUpdateReceivedOptions();
             return lobbyInterface.AddNotifyLobbyUpdateReceived(ref options, null, notificationFn);
         }
 
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyMemberUpdateReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-member-update-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
         private ulong AddNotifyLobbyMemberUpdateReceived(LobbyInterface lobbyInterface, OnLobbyMemberUpdateReceivedCallback notificationFn)
         {
             var options = new AddNotifyLobbyMemberUpdateReceivedOptions();
             return lobbyInterface.AddNotifyLobbyMemberUpdateReceived(ref options, null, notificationFn);
         }
 
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLobbyMemberStatusReceived](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-lobby-member-status-received)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
         private ulong AddNotifyLobbyMemberStatusReceived(LobbyInterface lobbyInterface, OnLobbyMemberStatusReceivedCallback notificationFn)
         {
             var options = new AddNotifyLobbyMemberStatusReceivedOptions();
             return lobbyInterface.AddNotifyLobbyMemberStatusReceived(ref options, null, notificationFn);
         }
 
+        /// <summary>
+        /// Use to access functionality of [EOS_Lobby_AddNotifyLeaveLobbyRequested](https://dev.epicgames.com/docs/api-ref/functions/eos-lobby-add-notify-leave-lobby-requested)
+        /// The callback will only run if a listener is subscribed, which is done in <see cref="SubscribeToLobbyUpdates"/>.
+        /// </summary>
+        /// <param name="lobbyInterface">Handle to the lobby interface.</param>
+        /// <param name="notificationFn">Callback to receive notification when lobby member update is received</param>
```

Comment:
Similar to comment blocks before: the notificationFn comment is incorrect, and comment describing the return value is missing.

Reply:
Return value added, changed it to "when lobby member leave request is received."